model_zoo/app_integrate_bundle/ensemble.py

```diff
@@ -220,21 +211,28 @@ def __call__(self, args, datalist, test_datalist=None):
                     "--meta_file",
                     self.bundle_metadata_path,
                     "--config_file",
-                    f"['{train_path}','{multi_gpu_train_path}']",
+                    f"['{self.bundle_config_path}','{multi_gpu_train_path}']",
                     "--logging_file",
                     self.bundle_logging_path,
+                    "--bundle_root",
+                    self.bundle_path,
+                    "--epochs",
+                    str(max_epochs),
+                    "--dataset_dir",
+                    dataset_dir,
+                    "--train#dataset#data",
+                    f"%{train_datalist_path}",
+                    "--validate#dataset#data",
+                    f"%{val_datalist_path}",
                 ]
                 self.run_command(cmd, env)
             else:
-                monai.bundle.run(
-                    meta_file=self.bundle_metadata_path,
-                    config_file=self.bundle_config_path,
-                    logging_file=self.bundle_logging_path,
-                    **overrides,
-                )
-            fold += 1
+                self.train_workflow.run()
```

Comment:
Please execute `self.train_workflow.finalize()` after all the running, same for `inference_workflow`